.fossa.yml

```diff
@@ -94,15 +94,3 @@ analyze:
     type: go
     target: github.com/m3db/m3/src/cmd/tools/verify_index_files/main
     path: src/cmd/tools/verify_index_files/main
-  - name: github.com/m3db/m3/src/query/benchmark/benchmarker/main
-    type: go
-    target: github.com/m3db/m3/src/query/benchmark/benchmarker/main
-    path: src/query/benchmark/benchmarker/main
-  - name: github.com/m3db/m3/src/query/benchmark/read/main
-    type: go
-    target: github.com/m3db/m3/src/query/benchmark/read/main
-    path: src/query/benchmark/read/main
-  - name: github.com/m3db/m3/src/query/benchmark/write/main
-    type: go
-    target: github.com/m3db/m3/src/query/benchmark/write/main
-    path: src/query/benchmark/write/main
```

Comment:
This may need to have the dependencies for the little test utilities e.g. in prom_comparator added?

Reply:
Sure yeah, this is mainly declaring which binaries get validated by FOSSA.